src/robusta/integrations/msteams/msteams_msg.py

```diff
@@ -65,8 +65,9 @@ def write_title_and_desc(self, platform_enabled: bool, finding: Finding, cluster
             self.__write_to_entire_msg([block])
 
     @classmethod
-    def __build_msteams_title(cls, title: str, status: FindingStatus, severity: FindingSeverity) -> str:
-        return f"{status.to_emoji()} {status.name.lower()} - {severity.to_emoji()} {severity.name} - **{title}**"
+    def __build_msteams_title(cls, title: str, status: FindingStatus, severity: FindingSeverity, add_silence_url: bool) -> str:
+        status_str: str = f"{status.to_emoji()} {status.name.lower()} - " if add_silence_url else ""
```

Comment:
Did you test the silence url works on teams, telegram and mattermost after the fix?

Reply:
Yup, tested across all of them